### ISSUE TITLE
Computed values and connect

### ISSUE THREAD

**Issue by nd0ut** [cerebral/cerebral#1119]
Hi guys! I updated cerebral to newer version (with separated view packages) and get an error `Prop 'foo' should be a tag or a compute.` when I passed computed to the connect function.

I looked at the code of View.js and noticed that it checks property for getTag method and throws this error if it has not such method. So it means that only tags are allowed. It's planned or just a mistake?

**Comment by Guria** [comment]
Are your problem was solved?

**Comment by nd0ut** [comment]
@Guria Yes, but I do not know what was the matter, I just reinstalled all the dependencies with removing package.lock file and everything became ok 0_о

**Comment by Jhony-Reyes** [comment]
I had the same problem until I found nd0ut's answer, why does this happen?

**Comment by christianalfoni** [comment]
@nd0ut @Jhony-Reyes I believe this happens because of the versioning. NPM is confused by the datetime we add to the version. Though does this happen often? We did cause a version problem at some point, though that was resolved. But yeah, we need to version this way for now. I usually manually update these versions, clean out node_modules and do a fresh install

**Comment by Jhony-Reyes** [comment]
@christianalfoni @nd0ut This don't happen often, it's the first time that happens to me. I upgraded to `cerebral@next` two days ago and I had that problem until I found this issue. I deleted the package-lock.json and reinstalled the dependencies. Ready, this works!

**Comment by christianalfoni** [comment]
@Jhony-Reyes Ah yes, then it was very likely the transition issue we had :) Let me know if it pops up again and becomes problematic!
